maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -104742,6 +105094,14 @@
 /mob/living/bot/cleanbot/roomba,
 /turf/simulated/floor/tiled/white,
 /area/eris/medical/medbay)
+"rkA" = (
+/obj/machinery/button/remote/blast_door{
+	id = TechnomancerShopLockdown;
```

Comment:
```suggestion
	id =  "TechnomancerShopLockdown";
```

Reply:
for some reason this aint letting me commit this

Reply:
manually did it